src/saving/InProgressLoad.cs

```diff
@@ -69,6 +69,7 @@ private void Step()
             case State.Initial:
                 state = State.ReadingData;
                 LoadingScreen.Instance.Show(TranslationServer.Translate("LOADING_GAME"),
+                    MainGameState.Invalid,
```

Comment:
Could you add a comment before line 71 saying that MainGameState.Invalid is used here as that information is not known yet.
And a TODO comment here linking to that new issue you said you are going to create about putting the game state in the save info that could be used here.